website/docs/faqs/Git/gitignore.md

```diff
@@ -51,7 +53,9 @@ For more info on `gitignore` syntax, refer to the [Git docs](https://git-scm.com
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-ide/gitignore-italics.png" width="50%" title="A dbt project on the main branch that has properly configured gitignore folders (highlighted in italics)."/>
 
-### Fix in the git provider
+</Expandable>
+
+<Expandable alt_header="Fix in the git provider">
```

Comment:
I realize it's lower case elsewhere but I'll grab this one now and the rest post-holiday
```suggestion
<Expandable alt_header="Fix in the Git provider">
```

Reply:
good shout!